ccm_web/client/src/utils/handleErrors.ts

```diff
@@ -48,7 +50,7 @@ const handleErrors = async (resp: Response): Promise<void> => {
       console.error(text)
       throw new NotFoundError()
     default:
-      text = (JSON.parse(await resp.text()) as Error).message
+      text = JSON.stringify((JSON.parse(await resp.text()) as ErrorObj).errors)
```

Comment:
So, if the client needs to use this as data, you're thinking they would just `JSON.parse`? I think one alternative would be to create a new `Error` subclass, maybe `DefaultError` or `APIError` that assigns the error data to an instance variable (through the constructor).

Class could look something like this:
```
class DefaultError extends Error {
  public name = 'DefaultError'
  errors: SingleError[] // types as you have them now

  constructor (errors: SingleError[]) {
    super('Received one or more errors associated with this request from Canvas')
    this.errors = errors
  }
}
```

and then you'd do this:
```
    default:
      errors = (JSON.parse(await resp.text()) as ErrorObj).errors
      throw new DefaultError(errors)
```

And then Chris or FE developer could do this in React components:
```
if ( setCourseNameError instanceof DefaultError ) {
  setCourseNameError.errors ...
}
```

Reply:
I am going by this approach

Reply:
Done